source/_components/emulated_hue.markdown

```diff
@@ -90,26 +92,17 @@ emulated_hue:
   expose_by_default: true
   exposed_domains:
     - light
-```
-
-With additional customization you will be able to specify the behavior of the existing entities.
-
-```yaml
-# Example customization
-homeassistant:
-  customize:
+  entities:
     light.bedroom_light:
-      # Don't allow light.bedroom_light to be controlled by the emulated Hue bridge
-      emulated_hue_hidden: true
-    light.office_light:
-      # Address light.office_light as "back office light"
-      emulated_hue_name: "back office light"
+      name: "Bedside Lamp"
+    light.ceiling_lights:
+      hidden: true
 ```
 
-The following are attributes that can be applied in the `customize` section:
+The following are attributes that can be applied in the `entities` section:
```

Comment:
Should we keep the old `customize` section but mention that the `entities` attribute is the preferred way of doing things?

Reply:
Yeah, let's add a single line at the bottom that this used to be part of customize but that we're moving to have config under `entitities`